sdk/contentsafety/ai-content-safety-rest/samples-dev/sampleAnalyzeText.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates how to analyze text.
+ */
+
+import ContentSafetyClient, {
+  AnalyzeTextParameters,
+  AnalyzeTextOptions,
+  isUnexpected
+} from "@azure-rest/ai-content-safety";
+import { AzureKeyCredential } from "@azure/core-auth";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+async function main() {
+  const endpoint = process.env["CONTENT_SAFETY_ENDPOINT"] || "<endpoint>";
+  const key = process.env["CONTENT_SAFETY_API_KEY"] || "<key>";
+
+  const credential = new AzureKeyCredential(key);
+  const client = ContentSafetyClient(endpoint, credential);
+
+  const text = "You are an idiot";
```

Comment:
I don't think I love the idea of having offensive content here. Can we put a placeholder with something like "<Replace with potentially offensive string">?

Reply:
Or can we use the inverse sample where we test a non-offensive string?

Reply:
For samples, I think it's okay to use a placeholder here, but for test, maybe we should use the inverse samples.

Reply:
Replaced the text string with "This is a sample text". 
We want SDK users to run the sample code directly, so we will try to avoid using placeholders in the sample.